columnflow/calibration/cms/jets.py

```diff
@@ -111,58 +115,130 @@ def ak_evaluate(evaluator: correctionlib.highlevel.Correction, *args) -> float:
 #
 
 # define default functions for jec calibrator
-def get_jec_file_default(self, external_files: DotDict) -> str:
+def get_jerc_file_default(self, external_files: DotDict) -> str:
     """
-    Function to obtain external jec files.
+    Function to obtain external correction files for JEC and/or JER.
 
     By default, this function extracts the location of the jec correction
-    files from the current config instance *config_inst*:
+    files from the current config instance *config_inst*. The key of the
+    external file depends on the jet collection. For ``Jet`` (AK4 jets), this
+    resolves to ``jet_jerc``, and for ``FatJet`` it is resolved to
+    ``fatJet_jerc``.
```

Comment:
I think an all-lowercase `fatjet_jerc` key would be more consistent.
Would you mind renaming? (same for the occurrences below)

Reply:
Renamed to `fat_jet_jerc` as discussed privately.